package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Schedule processing utilities",
   "main": "./lib/index.js",
   "scripts": {
-    "build": "babel src --out-dir lib && webpack",
+    "build": "tsc && webpack",
```

Comment:
Builds still go into `lib` (for package consumption) and `dist`(for bundle consumption), AFAIK the usage should be the exact same except we have typings now. Full directory listing for `dist` and `lib`:

```bash
leonli@Leons-MBP:schedule-utilities (leontastic/feat/typescript)$ ls dist/** lib/**
dist/schedule-utilities.js		lib/processOverrides.test.js
lib/index.d.ts				lib/processOverrides.test.js.map
lib/index.js				lib/recurrenceIterator.d.ts
lib/index.js.map			lib/recurrenceIterator.js
lib/index.test.d.ts			lib/recurrenceIterator.js.map
lib/index.test.js			lib/recurrenceIterator.test.d.ts
lib/index.test.js.map			lib/recurrenceIterator.test.js
lib/processOverrides.d.ts		lib/recurrenceIterator.test.js.map
lib/processOverrides.js			lib/rrule-alt.d.ts
lib/processOverrides.js.map		lib/rrule-alt.js
lib/processOverrides.test.d.ts		lib/rrule-alt.js.map

lib/types:
Frequency.d.ts		Occurrence.js.map	WeekDay.js
Frequency.js		Sequence.d.ts		WeekDay.js.map
Frequency.js.map	Sequence.js		index.d.ts
Occurrence.d.ts		Sequence.js.map		index.js
Occurrence.js		WeekDay.d.ts		index.js.map
```

Reply:
I don't think we use `dist` for anything. I'm in favour or just removing it as it should be recommended to consumers to use the `lib` version and use webpack or another bundler at the app-level.

Reply:
@jschr Sure I can remove it